app/src/main/java/mozilla/lockbox/LockboxApplication.kt

```diff
@@ -46,13 +48,16 @@ class LockboxApplication : Application() {
         // use context for system service
         ClipboardStore.shared.apply(getSystemService(Context.CLIPBOARD_SERVICE) as ClipboardManager)
         FingerprintStore.shared.apply(getSystemService(Context.FINGERPRINT_SERVICE) as FingerprintManager)
+        LockedStore.shared.apply(getSystemService(Context.KEYGUARD_SERVICE) as KeyguardManager)
 
         // use context for PreferenceManager
         SecurePreferences.shared.apply(PreferenceManager.getDefaultSharedPreferences(this))
 
         // hook this context into Telemetry
         TelemetryStore.shared.applyContext(this)
 
+        FingerprintStore.shared.applyContext(this)
```

Comment:
can we set up our `lateinit` properties (such as `FingerprintManager`, `KeyguardManager`) in this `applyContext` method instead of instantiating them above?

Reply:
^ that would obviate the need for the `apply` method in the `FingerprintStore` which would be awesome